.github/workflows/pull-unit-test.yaml

```diff
@@ -0,0 +1,21 @@
+name: pull-unit-test
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+    types: [opened, reopened, synchronize, ready_for_review, converted_to_draft]
+  workflow_dispatch:
+jobs:
+  unit-tests:
+    name: unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: Setup Go
```

Comment:
Could we unify setup-go in all our projects?

Reply:
https://github.com/kyma-project/serverless/issues/1034